apps/src/music/views/SoundsPanel.tsx

```diff
@@ -286,6 +289,27 @@ const SoundsPanel: React.FunctionComponent<SoundsPanelProps> = ({
     );
   }
 
+  const showFilterTypes: {[key: string]: boolean} = {
+    all: true,
+  };
+
+  allSoundEntries.forEach(soundEntry => {
```

Comment:
Could we do this calculation up front as soon as we have the library loaded? That would be less logic in this component and might be helpful to have this information accessible elsewhere too?

Reply:
Good idea; done.